_build/html/_sources/index.txt

```diff
@@ -1,83 +1,144 @@
-..
-  Content of technical report.
 
-  See http://docs.lsst.codes/en/latest/development/docs/rst_styleguide.html
-  for a guide to reStructuredText writing.
+:tocdepth: 2
 
-  Do not put the title, authors or other metadata in this document;
-  those are automatically added.
+Introduction
+============
 
-  Use the following syntax for sections:
+In order to test Differential Chromatic Refraction (DCR) correction algorithms we first need a metric that quantifies the severity of residual dipoles in difference images, in order to evaluate the performance of the correction. 
+To begin, I used the StarFast simulator (http://dmtn-012.lsst.io, https://github.com/lsst-dm/starfast_simulator) to generate multiple realizations of a couple small fields at u, g, and r bands and across a range of airmasses from 1.0 to 2.0.
+These simulated images contain a realistic distribution of different stellar types, and using several thousand different model SEDs (interpolated Kurucz models, from sims_photUtils).
```

Comment:
@jonathansick may correct me but when you mention LSST software packages you can use:
```
:lmod:`sims_photUtils`

and then later it will be trivial to turn that into a link to the package documentation.